tests/fast-integration/test/fixtures/commerce-mock/index.js

```diff
@@ -206,9 +194,70 @@ async function sendEventAndCheckResponse(mockNamespace = 'mocks') {
   );
 }
 
-async function sendLegacyEventAndCheckTracing(targetNamespace = 'test', mockNamespace = 'mocks') {
-  // Send an event and get it back from the lastorder function
-  const res = await sendEventAndCheckResponse(mockNamespace);
+async function sendLegacyEventAndCheckResponse(mockNamespace = 'mocks') {
+  const body = {
+    'event-type': 'order.created',
+    'event-type-version': 'v1',
+    'event-time': '2020-09-28T14:47:16.491Z',
+    'data': {'orderCode': '567'},
+    'event-tracing': true,
```

Comment:
Without this field the tracing tests aren't working

Reply:
@VladislavPaskar What exactly is not working ? I am curious and just asking because I wonder if this is something which is important for the customer as well 🤔 
If it is some test internal think, let's don't care about it and continue. If this is required somehow for tracing to be working, then we should probably document it 🤔 

I searched inside the kyma code base and don't find it anywhere else than the fast-integration tests.

Reply:
I didn't find where this is used as well, but certainly I know that it fails as soon as you remove it. We can watch together, what does it mean. But figuring it out IMO is out of scope of this ticket. As it was already there and was just refactored to a separate function/

Reply:
It comes from varkes, when you pass the `event-tracing` parameter, varkes sets the ` x-b3-sampled` header which disables for the call to Kyma, causing the istio-proxies to collect traces no matter what trace sampling rate is configured for istio.

Reply:
@VladislavPaskar thanks a lot for the explanation :) So this is totally not facing the customer 👍

Reply:
Also 👍 for adding a comment. That will probably avoid future confusion :)